sfge/src/test/java/com/salesforce/rules/unusedmethod/OverloadsTest.java

```diff
@@ -319,7 +317,7 @@ public void callConstructorViaThisWithDifferentArityOverloads_expectViolation(
                         + String.format("    %s MyClass(boolean b, boolean c) {\n", scope)
                         + "    }\n"
                         // Use the engine directive to prevent this method from tripping the rule.
-                        + "    /* sfge-disable-stack UnusedMethodRule */"
+                        + "    /* sfge-disable-stack UnusedMethodRule */\n"
```

Comment:
The missing newline meant that this directive was actually on the same line as the constructor and therefore wasn't being applied. Since public constructors weren't supported yet, that caused no problems. Adding support for public constructors caused the tests to start failing, and adding the newline fixes that.